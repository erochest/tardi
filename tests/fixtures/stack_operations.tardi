1 2 3 dup
4 5 swap
6 7 8 rot
9 drop
