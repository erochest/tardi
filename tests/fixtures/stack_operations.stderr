1
2
3
3
5
4
7
8
6
